sneakers/tests/test_encoders/rsa_test_keys/test_key

```diff
@@ -0,0 +1,27 @@
+-----BEGIN RSA PRIVATE KEY-----
+MIIEpAIBAAKCAQEA7QjVvebIc5OsmbvSWfArNK8Rfo260WrDX8IrM/n9L1CEMuOM
+l00RMSbHGtLPMWVHdXGJ+Hmy239DuNyRgrhB5OrfFFYzdWNTrn0i0Ut9XDbPTfhX
+o0vFQo85MpjwaOmFswjWBaFyp9v9L/l1MXax1tiB0KRzvB+O1MyUKaG73mixoMC5
+qG4BlSB4PIZzlrpQLChhZ6bG1DG4kDIcHG1+PelaERkUTQ95NE01U5nmgEU4n4b8
+V+6mBP599tU4PG+lR4BMsGWhwlI0/E0ngeSbUgBPp/SMCZXbzMUUF/DNGkcrJVmE
+H+6ObpKCDKPDuxwrAr5ABwLMBLRh6CFF+iIIqwIDAQABAoIBAQDqvOMfq1s7yjUW
+EgEg/E1tYSqMD8PA/F/7hwjN8nYf0zb4RozGkZXC/UoUcEyr+EaqMgSWw76yzPe+
+LQEVWXA4SbkeJLfVOsZ2eIpwhrfhCI6IAraXisQRRFE6oGcFHq4/PTLmZJptTdkR
+F5RNpeQ8hjEY9KOA3CfQ6cCY8ghhGZlWptRWu+NHF1DPLi8C4HNuUFPp3iNE1dra
+BwjrUNjY5PSUM9xsBU/Vq3kW6sfoBhrAuDbhZ+EZ0tU7CayZukg9vfN/veo6uBTS
+/tcPA/IGXN5OdVf/8nrS63UzkdW6tplEzjbbzIkLJ2oGdQ/3bvAFLS6VALZyhgdn
+ISECx9UxAoGBAPd2fYZAyRdNIFYxl4SBvtKUrZt2wairH6r7gSk7MfRNv4pmNBSd
+UTTwxZ4DTsWpk4vP/7JSUU5Evym5A/hEidPPv5tw1J7FsiuciRafdNB7OtklbO7T
+FrRQsG5ZJxvm7W9v6hbexOAW1DUg7msjdPlrM3KqkHr3FJ13s2bpBVFNAoGBAPU2
+PrVJvqP66qLnkl2juYnzUnflaqOy0A5y9YXCpAKBArtpksl1kHLwI8NS5bTWavr0
+OzgHbxaxbK+D/iN//ftXbnRXB7T4yDVRhR7xZdcm9Sf0qJx5YE5Q2UylGQhkvblc
+NvvOL4oCEaoy2G8Q0FoRrA/5O09gO+EiVe+mRUXXAoGAB5209UnPJpFwGlcCY8ie
+d9qtkShRg4BoxBlje0JgIjW2XGsS1UOWOTvjyR4u8XRrnxTyuKlB6VfCTmVsUtrB
+Jyywp/cMBCrvddomefTEYpo4UHCPbrY+b6RAGdVSl9A+pHxcRwHqKz9ono356IA5
+3GdJORE4yyAJmWMSuWsYh80CgYEAg4rvBuehmmwfdPaPQSZ0gPmD8weiNmDVVfr7
+lR7+Rs2Yfps7AFHOBKqoNL8CtQkVaLyeDAeI436YlGtVaQvfXvbfLRAG9WFEXb32
+kMvk2w9uElogHZjLPhgHCiPDEwqf8JD0eg542CsV+Ny/2I0HiFnUmNkVpRY16Sc3
+PJFQtg8CgYAeiiVyPfVahaSkdMko3X6i2kAZ07x5OMJKUiejyp0GRWCPG/dJ2yiv
+M5ZgSuK7T6u8Nh/nwnzo3LaGCvFlNLlNeROoziKq/PSdNjjYwDbUJBb35HdhSxOT
+hQjYZIS6snqfE0pSaLoDnlqi34gsiTyXunHeQet1cSizIeGOtSxQHw==
+-----END RSA PRIVATE KEY-----
```

Comment:
This would ideally go in `/sneakers/config/`, but it's not that important - just keeps things a little tidier.